release.sh

```diff
@@ -18,11 +18,13 @@ changelog() {
 
   echo "Generating changelog upto version: $NEXT"
   github_changelog_generator \
+    --user="leapfrogtechnology" \
+    --project="sync-db" \
+    --token="$GITHUB_TOKEN" \
     --no-verbose \
     --pr-label "**Changes**" \
     --bugs-label "**Bug Fixes**" \
     --issues-label "**Closed Issues**" \
-    --issue-line-labels=ALL \
```

Comment:
@kabirbaidhya @mesaugat i was getting this error locally 
![Screenshot from 2021-03-09 17-37-29](https://user-images.githubusercontent.com/35683880/110467508-36d6e180-80ff-11eb-8372-53e7a1357300.png)
says notsuchmethod so i removed it. does it affect anything ?

Reply:
It should be fine.

Reply:
We lost `labels` from the changelog but it seems to be better in my opinion. Feature not a bug.

Reply:
OK resolving